src/functions/genfrac.js

```diff
@@ -166,8 +166,10 @@ defineFunction({
                 positionType: "individualShift",
                 children: [
                     {type: "elem", elem: denomm, shift: denomShift},
-                    // $FlowFixMe `rule` cannot be `null` here.
-                    {type: "elem", elem: rule,   shift: midShift},
+                    // The next line would ordinarily contain "shift: midShift".
+                    // But we put the rule into a a span that is 5 rules tall,
+                    // to overcome a Chrome rendering issue.
+                    {type: "elem", elem: rule,   shift: midShift + 2 * ruleWidth},
```

Comment:
It might be good to further elaborate on the comment by stating that the top/bottom padding is 2 spans in height which corresponds to the `2 * ruleWidth` we're adding to the `shift`.

Reply:
Done.